config/step/ot_ld_based_clumping.yaml

```diff
@@ -1,7 +1,7 @@
 defaults:
   - ld_based_clumping
 
-ld_index_path: ${datasets.ld_index}
+ld_index_path: ${datasets.ld_index}/2.1.1
```

Comment:
I have regenerated all gnomAD related steps.  This includes ot_ld_index and ot_variant_annotation.  The first one fetches the raw ld matrices from gnomAD for given populations and creates the ld_index in staging bucket (XX.XX).

These steps were run because there is additional ld matrix for est(Estonian population) which was missing in our original ld_index populations list. -> this resulted in slightly bigger ld_index (66GB -> 74GB). We could save it in the static assets, but I decided to add the versioning for this data source, so it exists in the staging bucket along with the original ld_index, under the 2.1.1 directory. 

Other step was run because gnomAD introduced version 4.1 for variant annotation source.  

Was there additional curation done outside of the ot_ld_index step done to the static asset you refer to ?

Reply:
Thanks for the explanation! If the data is good I think it's clearer if the newer LD index replaced the one in the static_assets, or if we added the versioning there. In the end this is the location that we are going to share with partners.
In general, I'd prefer not to have the configuration for the staging phase merged into dev, which could be promoted to main.

Reply:
Sure thing! I will update the paths and shuffle the data in the static assets to match versioning.